benchmark/_test-double-benchmarker.js

```diff
@@ -1,14 +1,19 @@
 'use strict';
 
-const http = require('http');
+const myModule = process.argv[2];
+if (!['http', 'http2'].includes(myModule)) {
+  throw new Error(`Invalid module for benchmark test double: ${myModule}`);
+}
+
+const http = require(process.argv[2]);
```

Comment:
It might sound silly but I'd prefer:
```suggestion
const http = myModule === 'http'? require('http') : require('http2');
```
It makes it easier to later do static analysis (even a simple `grep`)

Reply:
My thinking was that this might not be the last time we extend this for benchmarking. For example, I could imagine a `tls`-specific benchmark test double.

Reply:
Maybe 
```suggestion
const http = require(myModule);
```
?